crates/rooch/src/commands/session_key/commands/create.rs

```diff
@@ -36,6 +41,15 @@ impl CreateCommand {
     pub async fn execute(self) -> RoochResult<SessionKey> {
         let mut context = self.context_options.build().await?;
 
+        let password = if self.password_required == Some(false) {
+            // Use an empty password if not required
+            String::new()
+        } else {
+            // Prompt for a password if required
+            rpassword::prompt_password("Enter a password to encrypt the keys in the rooch keystore. Press return to have an empty value: ").unwrap()
+        };
+        println!("Your password is {}", password);
```

Comment:
Do not print the password when decrypting.

Reply:
I leaved it with empty string for a new password.